src/main/java/com/gluonhq/substrate/util/ios/CodeSigning.java

```diff
@@ -243,6 +246,15 @@ private boolean sign(Path entitlementsPath, Path appPath) throws IOException, In
             Logger.logSevere("Codesign process failed");
             return false;
         }
+        for (String line : runner.getResponses()) {
+            if (line.contains(KEYCHAIN_ERROR_MESSAGE)) {
+                Logger.logInfo("Error signing the application: keychain was locked");
```

Comment:
That sounds like the operation is halted, but from the code it seems you're continuing trying to solve this.
It would be good if the developer got feedback (e.g. "You will know be shown a dialog" or so)

Reply:
Makes sense, modified.